hrm-domain/hrm-service/src/case/sql/case-search-sql.ts

```diff
@@ -17,9 +17,9 @@
 import { pgp } from '../../connection-pool';
 import { SELECT_CASE_SECTIONS } from './case-sections-sql';
 import { CaseListFilters, DateExistsCondition, DateFilter } from '../case-data-access';
-import { leftJoinCsamReportsOnFK } from '../../csam-report/sql/csam-report-get-sql';
-import { leftJoinReferralsOnFK } from '../../referral/sql/referral-get-sql';
-import { leftJoinConversationMediasOnFK } from '../../conversation-media/sql/conversation-media-get-sql';
+import { selectCoalesceCsamReportsByContactId } from '../../csam-report/sql/csam-report-get-sql';
+import { selectCoalesceReferralsByContactId } from '../../referral/sql/referral-get-sql';
+import { selectCoalesceConversationMediasByContactId } from '../../conversation-media/sql/conversation-media-get-sql';
```

Comment:
are the old exported functions being used anywhere else?
- `leftJoinCsamReportsOnFK`
- `leftJoinReferralsOnFK`
- `leftJoinConversationMediasOnFK`

looks like they can be removed.

Reply:
You're right, doing that